test/transaction-runner.ts

```diff
@@ -229,11 +231,37 @@ describe('TransactionRunner', () => {
         await runner.getTransaction();
         assert.strictEqual(beginStub.callCount, 1);
       });
+
+      describe('when multiplexed session is enabled for read/write transaction', () => {
+        it('should set the multiplexedSessionPreviousTransactionId in the new transaction object', async () => {
+          const expectedTransaction = new FakeTransaction();
+          const fakePreviousTransactionId = 'fake-transaction-id';
+          sandbox.stub(expectedTransaction, 'begin').resolves();
+
+          sandbox.stub(SESSION, 'transaction').returns(expectedTransaction);
+          delete runner.transaction;
+
+          runner.multiplexedSessionPreviousTransactionId =
+            fakePreviousTransactionId;
```

Comment:
Why are you setting this here, this should be set by code ?

Reply:
How would we test if the `multiplexedSessionPreviousTransactionId` is set correctly in runner ?